cnxeasybake/oven.py

```diff
@@ -146,12 +154,47 @@ def update_css(self, css_in=None, clear_css=False):
             except AttributeError:
                 css = css_in         # Treat it as a string
 
+        # Namespaces defined in the CSS
+        if clear_css:
+            self.css_namespaces = {}
+
         rules, _ = tinycss2.parse_stylesheet_bytes(css, skip_whitespace=True)
         for rule in rules:
-            # Ignore all at-rules
-            if rule.type == 'qualified-rule':
-
-                selectors = parse(rule.prelude, extensions=extensions)
+            # Check for any @namespace declarations
+            if rule.type == 'at-rule':
+                if rule.lower_at_keyword == 'namespace':
+                    # The 1 supported format is:
+                    #
+                    # default (unsupported):  [<WhitespaceToken>,
+                    #            <StringToken "http://www.w3.org/1999/xhtml">]
+                    # prefixed: [<WhitespaceToken>,
+                    #            <IdentToken html>,
+                    #            <WhitespaceToken>,
+                    #            <StringToken "http://www.w3.org/1999/xhtml">]
+                    if ((len(rule.prelude) == 4 and
```

Comment:
I added an extra parentheses because pep8 gave me this error:

```
cnxeasybake/oven.py:173:5: E129 visually indented line with same indent as next logical line
```

and when I tried to in/outdent `pep8` complained about being under/over indented so I gave up and just added an extra paren. Is there a better way to fix this?